src/bindings/python/src/pyopenvino/frontend/manager.cpp

```diff
@@ -76,15 +77,19 @@ void regclass_frontend_FrontEndManager(py::module m) {
 
     fem.def(
         "load_by_model",
-        [](const std::shared_ptr<ov::frontend::FrontEndManager>& fem, const std::string& model_path) {
-            return fem->load_by_model(model_path);
+        [](const std::shared_ptr<ov::frontend::FrontEndManager>& fem, const py::object& model) {
+            if (py::isinstance(model, py::module_::import("pathlib").attr("Path"))) {
+                std::string model_path = Common::utils::convert_path_to_string(model);
+                return fem->load_by_model(model_path);
+            }
+            return fem->load_by_model({Common::utils::py_object_to_any(model)});
```

Comment:
can we use just this line? string is also wrapped into `ov::Any`

Reply:
81-83 lines perform converting pathlib.Path to string. If we remove it, then load_by_model( Path object) will not work. It does not work on master, so I added it here similar to fronten.load() binding: https://github.com/openvinotoolkit/openvino/blob/ee659c1ce8a70850e1490629ef56fcd48d716585/src/bindings/python/src/pyopenvino/frontend/frontend.cpp#L34